src/App.js

```diff
@@ -1,9 +1,11 @@
-
+import Display from './components/Display';
+import ButtonPanel from './components/ButtonPanel';
 
 function App() {
   return (
     <div className="App">
-      <h1>Hello</h1>
+      <Display />
+      <ButtonPanel />
```

Comment:
**REQUIRED**
It is placed in src/components directory ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/App.png))